components/brave_wallet_ui/assets/asset-icons/zrx-asset-icon.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Gotcha, did not run through any optimizer. I Just exported from Figma and reviewed, seemed like Figma did a pretty good job! But may be redundant to have both in there. Ultimately it would be awesome to have a lot of these icons in the brave-ui library in the future :)